### DISCUSSION TITLE
Property Tax Module Not Accepting PDF, PNG, or JPG File Formats

### DISCUSSION THREAD

**Discussion by KeertanaITL** [upyog/UPYOG#1114] [Q&A]
I'm having trouble with the Property Tax module on the UpYog platform. It seems that the module is not allowing me to upload files in PDF, PNG, or JPG formats. Here are the details:

I have tried uploading the following file types: .pdf, .png, and .jpg.
The files are within the allowed size limits and are properly formatted.
Each attempt results in a failure to upload the files, with no specific error messages indicating what went wrong.
Steps I’ve taken to troubleshoot:

Verified that the file input element accepts the correct formats.
Checked browser console for any error messages or logs.
Cleared browser cache and tried uploading again.
Ensured network connectivity and tested with different browsers.
Despite these efforts, I still can't upload any files in the Property Tax module.

Could this be an issue with the frontend code or possibly a backend configuration? Here are some specific questions:

Is there a known issue with file upload functionality in the Property Tax module?
Are there specific configurations or settings I need to adjust to enable file uploads?
Can someone provide guidance on where to check for potential issues, either in the frontend or backend?

**Comment by vinothrallapalli-eGov** [comment]
Please verify that the egov-filestore service is running and please refer to the link to configure allowed file formats,
https://github.com/upyog/UPYOG-Setup-DevOps/blob/e3f01bd7fc6d2ec82f54079050bff3b82005375e/config-as-code/environments/upyog-demo.yaml#L174-L175
